docs/guides/postinstall.md

```diff
@@ -1,22 +1,22 @@
 # Introduction
 
-This page explains how to perform a basic set up after installing Linux on a T2 Mac.
+This page explains how to perform a basic setup after installing Linux on a T2 Mac.
 
 ## Do you need to do this?
 
-This guide is mainly relevant in the following cases :
+This guide is mainly relevant in the following cases:
 
 1. If you have installed Linux using an official ISO, instead of a T2 ISO.
 2. The [Make modules load on early boot](#make-modules-load-on-early-boot) section is relevant for those who wish to encrypt their disk drives using LUKS or some other similar software.
 3. If some functionality related to T2 Macs is broken, then you can consider following this guide.
 
-In rest cases, you probably won't need to follow this guide.
+In other cases, you probably won't need to follow this guide.
 
-If you have used a T2 ISO, **make sure you have followed the [distro specific guide](https://wiki.t2linux.org/distributions/overview/) for your distro before continuing further.**
+If you have used a T2 ISO, **make sure you have followed the [distro specific guide](https://wiki.t2linux.org/distributions/overview/) for your distro before continuing**.
 
 ## Installing a kernel for T2 support
 
-Installing a kernel with support for T2 Macs is required in order to get the Keyboard, Trackpad, Touch Bar, Audio, Fan and Wi-Fi working.
+Installing a kernel with support for T2 Macs is required to get the keyboard, trackpad, touch bar, audio, fan, and Wi-Fi working.
```

Comment:
Let's keep the capital letters here in order to kinda highlight what were are providing.